src/components/Accounts/Actions/CommentModal.test.js

```diff
@@ -0,0 +1,49 @@
+import { waitFor, screen, cleanup } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import renderWithRouter from 'helpers/renderWithRouter';
+import CommentModal from './CommentModal';
+
+jest.unmock('@folio/stripes/components');
+jest.unmock('@folio/stripes/smart-components');
+
+const renderCommentModal = (props) => renderWithRouter(<CommentModal {...props} />);
+
+const MockFunc = jest.fn();
+
+
+const propData = {
+  form: { reset: MockFunc },
+  pristine: false,
+  submitting: false,
+  onClose: MockFunc,
+  handleSubmit: MockFunc,
```

Comment:
More pedantic: don't re-use `MockFunc` here. Different attributes should receive different values, otherwise, we can't validate that they have been called correctly, i.e. the close-handler is supposed to submit the form and close the modal you don't have a way to test that here.